.github/workflows/pytest_e2e_pr_approval.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
```

Comment:
@sevenearths to run actions on PR approvals, the code should be:
on:
  pull_request_review:
    types: [submitted]

Reply:
Your right. I'll fix that